regtests/client/python/cli/options/option_tree.py

```diff
@@ -157,7 +160,8 @@ def get_tree() -> List[Option]:
                 ], input_name=Arguments.CATALOG),
                 Option(Subcommands.UPDATE, args=[
                     Argument(Arguments.CATALOG, str, Hints.CatalogRoles.CATALOG_NAME),
-                    Argument(Arguments.PROPERTY, str, Hints.PROPERTY, allow_repeats=True)
+                    Argument(Arguments.SET_PROPERTY, str, Hints.SET_PROPERTY, allow_repeats=True),
+                    Argument(Arguments.REMOVE_PROPERTY, str, Hints.REMOVE_PROPERTY, allow_repeats=True),
                 ], input_name=Arguments.CATALOG_ROLE),
                 Option(Subcommands.GRANT, hint=Hints.CatalogRoles.GRANT_CATALOG_ROLE, args=[
                     Argument(Arguments.CATALOG, str, Hints.CatalogRoles.CATALOG_NAME),
```

Comment:
namespaces changes are also missing here

Reply:
Right, for now namespaces don't actually have an `update` command yet (and never had one) so we can fill in missing features for namespaces in a separate PR